src/flag_gems/utils/triton_driver_helper.py

```diff
@@ -0,0 +1,8 @@
+try:
+    from ..runtime import torch_device_fn
+
+    get_device_properties = torch_device_fn.get_device_properties
```

Comment:
As backend/runtime utils getting chaotic, the code is getting harder to maintain. Can you define an interface class where individual backends can register their own instances?